modules/wayfire/hm.nix

```diff
@@ -0,0 +1,64 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+
+{
+  options.stylix.targets.wayfire.enable =
+    config.lib.stylix.mkEnableTarget "wayfire" true;
+
+  config =
+    lib.mkIf (config.stylix.enable && config.stylix.targets.wayfire.enable)
+      (
+        let
+          inherit (config.lib.stylix) colors;
+          rgb = rgb: ''\#${rgb}ff'';
+
+          wayfireCfg = config.wayland.windowManager.wayfire;
+
+          wayfireBackground = toString (
+            pkgs.runCommand "wayfire-background.png" { } ''
+              ${lib.getExe' pkgs.imagemagick "convert"} ${config.stylix.image} $out
+            ''
```

Comment:
> Wayfire currently isnt smart enough to understand what file type an image has if it doesnt have a file ending and we cannot depend on that being set.

This adds the `.png` extension to the `realpath` of the `/nix/store/<HASH>-wayfire-background.png` symlink. Does the `convert "$stylix_image" result` command actually do something? Also, does this work when `stylix.image` is not a PNG?